examples/dashboard-operator/cmd/manager/main.go

```diff
@@ -32,6 +33,7 @@ import (
 
 func main() {
 	addon.Init()
```

Comment:
Wondering if we should put flag.Parse() into addon.Init (and require that you declare flags before calling addon.Init.  Or alternatively we could require that you have called flag.Parse _before_ calling addon.Init - that might be easier to enforce.  But this shouldn't block this PR